qiling/os/posix/syscall.py

```diff
@@ -239,7 +239,17 @@ def ql_syscall_setuid(ql, null0, null1, null2, null3, null4, null5):
         UID = 1000
     ql.nprint("setuid(%i)" % UID)
     regreturn = UID
-    ql_definesyscall_return(ql, regreturn)     
+    ql_definesyscall_return(ql, regreturn)
+
+
+def ql_syscall_setpriority(ql, which, who, prio, null3, null4, null5):
+    ql.nprint("setpriority(%i %i %i)" % (which, who, prio))
+    try:
+        os.setpriority(which, who, prio)
```

Comment:
Dont have to set anything. return a 0 should be all fine

Reply:
Yeah you right, its not really necessary to call the actual syscall. 
I was trying to emulate Ruckus R510 and of course set-priority is not really matter.

Reply:
cool, we split the syscall file finally. so i guess you need to update the syscall. thanks !

Reply:
I'm not sure after the split where this change should be done.
./build/lib/qiling/os/posix/syscall.py or ./qiling/os/posix/syscall/unistd.py

Reply:
We split the posix/syscall.py file into multi-files in posix/syscall dir. The syscall should be in the file whose name is same with the header file which define the syscall fucntion. This case, setpriority is defined in resource.h,  ql_syscall_setpriority should be in syscall/resource.py. ql_syscall_clock_gettime should be in syscall/time.py.

Reply:
@waveburst maybe u can move this ql_syscall_setpriority to qiling/os/posix/syscall/resource.py ?